docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+services:
+  backend:
+    build: .
+    depends_on:
+      db:
+        condition: service_healthy
+    ports:
+      - 8080:5000
```

Comment:
Port 5000 on my machine was being used. Thus I have to map to port 8080 here.

You can try port 5000 on your machine and see what happens